encryptor/app.py

```diff
@@ -18,10 +19,10 @@
 class MainWindow(QMainWindow):
     """Main window of the application."""
 
-    def __init__(self, port: int, keys_dir: str) -> None:
+    def __init__(self, port: int, keys_dir: Path) -> None:
         super().__init__()
 
-        self._server_thread = ServerThread(Address("127.0.0.1", port))
+        self._server_thread = ServerThread(Address("127.0.0.1", port), keys_dir)
```

Comment:
Remove `keys_dir` from the server thread.

Reply:
done